### PR TITLE
Constraints - helper functions

### PR DESCRIPTION
**Pull request by FelipeAdachi**
## Description

This is a draft for simple constraints we might make available to the user to simplify constraints definition. 

## Changes

- Imperative commit or change title (commit ID) 
  - Add some bullet points to explain the change
- Next commit or change (commit ID)

## Related

Relates to organization/repo#number

Closes organization/repo#number

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: python/examples/advanced/Constraints_Suite.ipynb [hunk 1]
```diff
@@ -0,0 +1,3619 @@
...
+    "> Note: The constraints shown here are still experimental and subject to further changes. Stay tuned for upgrades!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from whylogs.core.constraints import ConstraintsBuilder, MetricConstraint, MetricsSelector"
```
Comment (TheMellyBee) [inline python/examples/advanced/Constraints_Suite.ipynb:94]: Need a %pip install whylogs before this to be able to run in colab

File: python/examples/advanced/Constraints_Suite.ipynb [hunk 2]
```diff
@@ -0,0 +1,3619 @@
...
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "builder = ConstraintsBuilder(dataset_profile_view=profile_view)\n",
```
Comment (TheMellyBee) [inline python/examples/advanced/Constraints_Suite.ipynb:1016]: Running in collab I got an error:
`Unable to load pybars; install pybars3 to load profile directly from the current session `

Do we need to make sure that's installed at the top?

**Review by TheMellyBee — COMMENTED** [review]
Over all I like this! It's mostly code which makes a lot of sense for what it is, *but* made it difficult to find what I wanted. Could a ToC be used at the top showcase where things are? 

Only need to install the two libraries to make it safe to run in Colab. :)

File: python/examples/advanced/Constraints_Suite.ipynb (hunk 1 above)
Comment (FelipeAdachi) [reply python/examples/advanced/Constraints_Suite.ipynb:94]: Thanks melly!
added pip install whylogs[viz], which solves the next comment about pybars3

File: python/examples/advanced/Constraints_Suite.ipynb (hunk 2 above)
Comment (FelipeAdachi) [reply python/examples/advanced/Constraints_Suite.ipynb:1016]: Yes, installing whylogs[viz] will install pybars
I forgot to add it, thanks!

Comment (FelipeAdachi) [reply python/examples/advanced/Constraints_Suite.ipynb:1016]: Also added TOC, related to the next comment (can't reply there, not sure why)

**Review by TheMellyBee — APPROVED** [review]
LGTM!

**Pull request merged**